### PR TITLE
Postal code

### PR DESCRIPTION
**Pull request by khooroko**
Add postal code field (will use postal code to group contacts by location)

### CODE REVIEW

**Review by lawwman — APPROVED** [review]
Addition of postal code field is good and follows the same coding style as the other fields of the person class.

File: src/main/java/seedu/address/model/person/Email.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import seedu.address.commons.exceptions.IllegalValueException;
 
 /**
- * Represents a Person's phone number in the address book.
+ * Represents a Person's email in the address book.
```
Comment (jelneo) [inline src/main/java/seedu/address/model/person/Email.java:8]: Unrelated code change for this PR. Could have done it in another PR

File: src/main/java/seedu/address/model/person/Email.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
     /**
      * Validates given email.
      *
-     * @throws IllegalValueException if given email address string is invalid.
+     * @throws IllegalValueException if given email email string is invalid.
```
Comment (jelneo) [inline src/main/java/seedu/address/model/person/Email.java:22]: Unrelated code change for this PR. Could have done it in another PR

**Review by jelneo — COMMENTED** [review]
Should keep the PR free of unrelated code change next time

**Review by jaivigneshvenugopal — APPROVED** [review]
Logic seems correct in your changes, since adding a field means to ensure that there should be an additional field in all the required classes and the tests. 

However, I just realised that there is no way to find a mistake or bug though. It's difficult to identify exactly where and which class requires editing.  Heavy regression testing using JUnit tests is the only way I guess. 

Great job man.
